cmd/fission-bundle/main.go

```diff
@@ -78,14 +78,26 @@ func runMessageQueueMgr(logger *zap.Logger, routerUrl string) {
 	}
 }
 
-func runStorageSvc(logger *zap.Logger, port int, filePath string) {
```

Comment:
- This is a typo. Function name will change back to `runStorageSvc` but with new signature. 
- This is a lot of logic here in `main.go`. I feel we can move this to a `Start()` method in storagesvc package. Just we are doing for others like `runTimer`, `runKubeWatcher` etc.